lib/index.js

```diff
@@ -192,14 +193,6 @@ parser.yy.parseError = function parseError(_, details) {
         throw err;
     }
 
-    if (details.text === '') {
```

Comment:
So the obvious question is, that since this is hard coded to return 'expected 'FROM'`, it's intended to catch the case where we were sent a rule with no content, what happens if we send an empty rule?

Turns out, a different part of the parser will catch this and supply a different error.

CloudAPI:

```
> triton fwrule create ''        
triton fwrule create: error (MissingParameter): rule is a required argument
```

`fwadm`:

```
[root@smartos-pb /var/tmp/bb/fw]# ./sbin/fwadm add -g -e ''
No rule specified
```

Getting more clever, trying a rule that consists of only a single space gives us the correct `expected: 'FROM'` error message as it is being handled by the next bit of code in the parser.

CloudAPI:

```
> triton -p barovia fwrule create ' '
triton fwrule create: error (InvalidParameters): Invalid parameters
    rule: InvalidParameter: Error at character 0: '', expected: 'FROM', found: EOF
```

`fwadm`:

```
[root@smartos-pb /var/tmp/bb/fw]# ./sbin/fwadm add --dryrun -g -e ' '
Error at character 0: '', expected: 'FROM', found: EOF
```